source/reference/methods.txt

```diff
@@ -730,6 +730,233 @@ Query Plan Cache Methods
        :manual:`plan cache entries </core/query-plans/>` 
        for a collection.
 
+Replication Methods 
+-------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+
+   * - :method:`rs.add()`
+
+     - Adds a member to the replica set. You must connect to the 
+       primary of the replica set to run this method.
+
+   * - :method:`rs.addArb()`
+
+     - Adds an arbiter to an existing replica set.
+
+   * - :method:`rs.config()`
+
+     - Returns a document that contains the current replica set configuration.
+
+   * - :method:`rs.freeze()`
+
+     - Makes the replica set member that ``mongosh`` is connected to 
+       ineligible to become primary for the specified duration. The 
+       duration must be specified in seconds.
+
+   * - :method:`db.getReplicationInfo()`
+
+     - Returns the status of the replica set from the oplog data.
+
+   * - :method:`rs.initiate()`
+
+     - Initializes a new replica set.
+
+   * - ``rs.isMaster``
+
+     - Returns the replica set configuration, role, and status for 
+       the ``mongod`` instance that ``mongosh`` is connected to. 
+       
+       The ``mongosh`` ``rs.isMaster()`` method wraps the server :dbcommand:`isMaster` command.
```

Comment:
`rs.isMaster()` here has parentheses. The reference above does not. Think they should be consistent. Not sure which one is correct.

Reply:
With parens is correct, I believe.